demos/common/mqtt_demo_helpers/mqtt_demo_helpers.c

```diff
@@ -478,20 +692,58 @@ static void vCleanupOutgoingPublishWithPacketID( uint16_t usPacketId )
 
 /*-----------------------------------------------------------*/
 
+static void prvUpdateSubAckStatus( MQTTPacketInfo_t * pxPacketInfo )
```

Comment:
Is it required to have this as a separate function if it has only a single caller within the `vHandleOtherIncomingPacket` function?